src/Rasterization/Renderers/SVGRectRenderer.php

```diff
@@ -21,25 +23,87 @@ protected function prepareRenderParams(SVGRasterizer $rasterizer, array $options
         $y1 = self::prepareLengthY($options['y'], $rasterizer) + $rasterizer->getOffsetY();
         $w  = self::prepareLengthX($options['width'], $rasterizer);
         $h  = self::prepareLengthY($options['height'], $rasterizer);
+        $rx = self::prepareLengthX($options['rx'], $rasterizer);
+        $ry = self::prepareLengthY($options['ry'], $rasterizer);
 
         return array(
             'x1' => $x1,
             'y1' => $y1,
             'x2' => $x1 + $w - 1,
             'y2' => $y1 + $h - 1,
+            'rx' => $rx - 1,
+            'ry' => $ry - 1,
         );
     }
 
     protected function renderFill($image, array $params, $color)
     {
+        $x1 = $params['x1'];
+        $y1 = $params['y1'];
+        $x2 = $params['x2'];
+        $y2 = $params['y2'];
+        $rx = $params['rx'];
+        $ry = $params['ry'];
+
+        if (($rx !== 0) || ($ry !== 0)) {
+            self::renderFillRounded($image, $params, $color);
+            return ;
+        }
         imagefilledrectangle(
             $image,
-            $params['x1'], $params['y1'],
-            $params['x2'], $params['y2'],
+            $x1, $y1,
+            $x2, $y2,
             $color
         );
     }
 
+    private function renderFillRounded($image, array $params, $color) {
+        $x1 = $params['x1'];
+        $y1 = $params['y1'];
+        $x2 = $params['x2'];
+        $y2 = $params['y2'];
+        $rx = $params['rx'];
+        $ry = $params['ry'];
+
+        imagefilledrectangle(
+            $image,
+            $x1 + $rx, $y1,
+            $x2 - $rx, $y2,
+            $color
+        );
+        imagefilledrectangle(
+            $image,
+            $x1, $y1 + $ry,
+            $x2, $y2 - $ry,
+            $color
+        );
```

Comment:
i found a problem of imagefilledarc.
```
$im = imagecreatetruecolor(600, 600);
imagesetthickness($im, 1);

imagefilledarc($im, 300, 300, 500, 530, 0,  90, 0x00ffff00, 0);
imagefilledarc($im, 300, 300, 500, 531, 90, 180, 0x00ff0000, 0);
imagefilledarc($im, 300, 300, 500, 532, 180, 270, 0x0000ff00, 0);
imagefilledarc($im, 300, 300, 500, 533, 270, 360, 0x000000ff, 0);
imagepng($im, "out.png");
```
![2017-08-24 22 39 29](https://user-images.githubusercontent.com/26040/29669168-8a20cc80-891d-11e7-91e3-c79bfb63445f.png)

"ry" odd number let the arc angle becomes insufficient . 
Continue to investigation. :-)